plugins/modules/df_service.py

```diff
@@ -337,6 +396,27 @@ def _wait_for_enabled(self):
             delay=self.delay, timeout=self.timeout
         )
 
+    def _filter_subnets(self, query, subnets):
```

Comment:
Because this function is used in multiple modules, it should probably be moved to a utility method accepting an array, a query, and a return field as a convenience method in common.py in cdpy similar to [_get_path](https://github.com/cloudera-labs/cdpy/blob/eeb420a791dc9f59ed84fe2e758e96bd10e954c0/src/cdpy/common.py#L449)

Reply:
Sounds good to me. I'll add the helper function to cdpy and I can improve this here in a separate change once the cpdy change is accepted and merged.